eanalyze_staging.sql

```diff
@@ -0,0 +1,1483 @@
+CREATE DATABASE  IF NOT EXISTS `eanalyze_staging` /*!40100 DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci */ /*!80016 DEFAULT ENCRYPTION='N' */;
+USE `eanalyze_staging`;

```

Comment:
We can remove these two lines. When recreating a database, we probably want to name it something other than eanalyze_staging anyway. Please also rename this file to create_database.sql?

Reply:
okay ,

Reply:
This is still in the script. Did you forget to push a change up to GitHub?